src/main/java/io/xpring/xrpl/ReliableSubmissionXRPClient.java

```diff
@@ -61,6 +64,11 @@ public RawTransactionStatus getRawTransactionStatus(String transactionHash) thro
         return this.decoratedClient.getRawTransactionStatus(transactionHash);
     }
 
+    @Override
+    public List<XRPTransaction> paymentHistory(String address) throws XpringException {
```

Comment:
Just a heads up that #174 renames `XpringException` to `XRPException` in case that lands before this one and you get build errors.